x/data/query.pb.gw.go

```diff
@@ -20,6 +20,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/grpclog"
+	"google.golang.org/grpc/metadata"
```

Comment:
why this file was modified?

Reply:
I'm not sure. Either leftover or my setup is slightly different.